lib/ansible/modules/cloud/amazon/ec2_asg.py

```diff
@@ -711,6 +727,11 @@ def get_properties(autoscaling_group):
     properties['termination_policies'] = autoscaling_group.get('TerminationPolicies')
     properties['target_group_arns'] = autoscaling_group.get('TargetGroupARNs')
     properties['vpc_zone_identifier'] = autoscaling_group.get('VPCZoneIdentifier')
+    raw_mixed_instance_object = autoscaling_group.get('MixedInstancesPolicy')
```

Comment:
This is a relatively new feature.  Do you know if a specific version of botocore is required for it?

Things like RHEL often have pretty old versions of boto3/botocore knocking about and it's nice to give them a clean 'You need version X of botocore' error message rather than some random exception.

Reply:
I think it was late 2018? But sorry, no idea in which version it was introduced.